test/realm/function-eval.js

```diff
@@ -12,7 +12,7 @@ test('function-no-body', t => {
 
 test('function-injection', t => {
   const goodFunc = 'return a+1';
-  const r = Realm.makeRootRealm();
+  const r = Realm().makeRootRealm();
```

Comment:
Do you really need each test to make their own `Realm` class? I think this shows that this repository should also provide the shim that wraps the polyfill, so that all the individual test cases can be phrased in the natural way that is more representative of expected user code, that will only make and install one shared `Realm` class.

Reply:
If we'd like to be super restrictive about isolating these unit tests we could create a class for each one but we do not need it. We'd have to create one for each test like
```
 import { makeRealmClass } from 'realm-shim';
 const Realm = makeRealmClass(...);
```

But if we keep exporting a default Realm we do not need any changes in our test files.

Reply:
That sounds good to me. The tests as originally written are more representative of code users would write.